pro/src/pages/Home/StatisticsDashboard/__specs__/OfferStats.spec.tsx

```diff
@@ -89,7 +89,7 @@ describe('OfferStats', () => {
 
   it('should render when the count is too high', async () => {
     vi.spyOn(api, 'getOffererV2Stats').mockResolvedValueOnce({
-      publishedPublicOffers: 10000,
+      publishedPublicOffers: 500,
```

Comment:
(détail) On risque d'afficher `500+` quand on a exactement 500 offres.
```suggestion
      publishedPublicOffers: 501,
```

Reply:
Techniquement on pourrait dire que 500+ inclus 500 :smile:

Reply:
Le back renvoie au plus `500` cf https://github.com/pass-culture/pass-culture-main/blob/88910ff2e9e0859b7155ecaeb3b8198dedfeac97/api/src/pcapi/core/offerers/repository.py#L717